server/src/main/java/com/objectcomputing/checkins/services/member_skill/skillsreport/SkillsReportServicesImpl.java

```diff
@@ -0,0 +1,191 @@
+package com.objectcomputing.checkins.services.member_skill.skillsreport;
+
+import com.objectcomputing.checkins.services.member_skill.MemberSkillRepository;
+import com.objectcomputing.checkins.services.memberprofile.MemberProfileRepository;
+import com.objectcomputing.checkins.services.member_skill.MemberSkill;
+import com.objectcomputing.checkins.exceptions.BadArgException;
+import com.objectcomputing.checkins.services.skills.SkillRepository;
+
+import javax.inject.Singleton;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+import java.util.Set;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.UUID;
+
+@Singleton
+public class SkillsReportServicesImpl implements SkillsReportServices {
+
+    private final MemberSkillRepository memberSkillRepo;
+    private final MemberProfileRepository memberProfileRepo;
+    private final SkillRepository skillRepo;
+    private final HashMap<String, Integer> skillLevels;
+
+    public SkillsReportServicesImpl(MemberSkillRepository memberSkillRepo,
+                                    MemberProfileRepository memberProfileRepo,
+                                    SkillRepository skillRepo) {
+        this.memberSkillRepo = memberSkillRepo;
+        this.memberProfileRepo = memberProfileRepo;
+        this.skillRepo = skillRepo;
+
+        skillLevels = new HashMap<>();
+        skillLevels.put("interested", 0);
+        skillLevels.put("novice", 1);
+        skillLevels.put("intermediate", 2);
+        skillLevels.put("advanced", 3);
+        skillLevels.put("expert", 4);
```

Comment:
You've already got this bit...why not go ahead and go full enum now? (bringing comment over from previous PR)

Reply:
I was assuming changing skill level to enum involves also updating `MemberSkill` and the related code since its skill level field is also `String`. So I was inclined to create another PR with that assumption. If converting to enum is only for this service, then I'll update this PR to do that. What do you think?

Reply:
Updated to use enum.